apps/jobs/serializers.py

```diff
@@ -122,7 +122,7 @@ def to_representation(self, instance):
             except Exception as err:
                 # We can also raise an exception here but this time, I am returning
                 # error message in the data
-                logger.error(f"An exception occurred: {err}")
+                self.logger.error(f"An exception occurred: {err}")
```

Comment:
Same message for the exception and log, log messages can contain in-depth stack trace and other messages if you want, but it should be the same.